README.md

```diff
@@ -18,3 +18,28 @@ My "to-do list" for this module:
 - should collect performance information (in writeable as csv)
 - command to view performance (https://github.com/codestudiohq/laravel-totem)
 - provide a debug toolbar pane
+
+## How to Try
+
+1. Add the following in your project's `composer.json`:
+
+```
+    "require": {
+        "codeigniter4/tasks": "dev-develop"
+    },
+```
+
+```
+    "repositories": [
+        {
+            "type": "vcs",
+            "url": "https://github.com/codeigniter4/tasks.git"
+        }
+    ],
+```
+
+2. Run `composer update`.
+
+3. Run `php spark migrate --all`.
+
+4. Copy `vendor/codeigniter4/settings/src/Config/Settings.php` into `app/Config/`.
```

Comment:
This will also need its namespace updated.

Reply:
Indeed. But I forgot to update the namespace, it seems to work.
`php spark tasks:list` shows correct task list.

Reply:
Added the instruction to update the namespace.